python/cudf/cudf/tests/test_binops.py

```diff
@@ -2913,3 +2913,52 @@ def test_empty_column(binop, data, scalar):
     expected = binop(pdf, scalar)
 
     utils.assert_eq(expected, got)
+
+
+@pytest.mark.parametrize(
+    "lhs",
+    [
+        pd.DataFrame([[1, 2, 3, 4], [5, 6, 7, 8]]),
+        pytest.param(
+            pd.DataFrame([[1, None, None, 4], [5, 6, 7, None]]),
+            marks=pytest.mark.xfail(
+                reason="Cannot access Frame.values if frame contains nulls"
+            ),
+        ),
+        pd.DataFrame([[1.2, 2.3, 3.4, 4.5], [5.6, 6.7, 7.8, 8.9]]),
+        pd.Series([14, 15, 16, 17]),
+        pd.Series([14.15, 15.16, 16.17, 17.18]),
+        [25.5, 26.6, 27.7, 28.8],
+    ],
+)
+@pytest.mark.parametrize(
+    "rhs",
+    [
+        pd.DataFrame([[9, 10], [11, 12], [13, 14], [15, 16]]),
+        pd.DataFrame([[9.4, 10.5], [11.6, 12.7], [13.8, 14.9], [15.1, 16.2]]),
+        pd.Series([5, 6, 7, 8]),
+        pd.Series([5.6, 6.7, 7.8, 8.9]),
+    ],
+)
+def test_binops_dot(lhs, rhs):
+    if isinstance(lhs, list):
+        glhs = lhs
+    else:
+        glhs = cudf.from_pandas(lhs)
```

Comment:
```suggestion
    glhs = lhs if isinstance(lhs, list) else cudf.from_pandas(lhs)
```
Should we be supporting dot between a pandas object and a cudf object? If so, perhaps you should be injecting this logic into `dot` rather than here. Same goes for `rhs` below (of course, at least one operand must be a `cudf` object to trigger our logic).

Reply:
You're right, we should be supporting dot between pandas and cuDF - I'll add that in and make some tests using pandas dataframes.